### ISSUE TITLE
Wrong dart path in some (older) flutter versions

### ISSUE THREAD

**Issue by Mantas0726** [leoafarias/fvm#348]
```
fvm doctor

FVM Version: 2.2.3
___________________________________________________

FVM config found:
___________________________________________________

Project: autobleidas
Directory: C:\flutter_projects\autobleidas
Version: 1.17.0
Project Flavor: None selected
___________________________________________________

Version is currently cached locally.

Cache Path: C:\Users\Mantoska\fvm\versions\1.17.0
Channel: false
SDK Version: 1.17.0

IDE Links
VSCode: .fvm/flutter_sdk
Android Studio: C:\flutter_projects\autobleidas\.fvm\flutter_sdk


Configured env paths:
___________________________________________________

Flutter:
C:\flutter\bin\flutter.bat

Dart:
C:\flutter\bin\dart.bat

FVM_HOME:
not set
```

**Describe the bug**

It points to a non-existing file when trying to execute dart.

**To Reproduce**

1. Go to terminal..
2. Run `fvm use 1.17.0` install if not installed.
3. run `fvm dart --version`
4. Should get 
```
fvm: running Dart from Flutter "1.17.0"

ProcessException: The system cannot find the file specified.

  Command: C:\Users\Mantoska\fvm\versions\1.17.0\bin\dart.bat --version
```

**Expected behavior**
It should reference dart in 1.17.0\flutter_sdk\bin\cache\dart-sdk

**Desktop (please complete the following information):**

- Windows 10
- FVM version 2.2.3
- Cmder, Android studio terminal, Command Prompt

**Comment by leoafarias** [comment]
@Mantas0726 Do you want to try a PR for this so I can review?

**Comment by callmephil** [comment]
I'm having a similar issue, When I try to install pubs on my old project.
When Running fvm dart in the terminal. I get this output:

```
phil@Pm app-app-flutter % fvm dart
fvm: running Dart from Flutter "v1.9.2"

Cannot execute /Users/phil/fvm/versions/v1.9.2/bin/dart

Please run command with  --verbose if you want more information
```

When running fvm Doctor:

```
FVM Version: 2.2.4
___________________________________________________

FVM config found:
___________________________________________________

Project: app-app-flutter
Directory: /Users/phil/Desktop/app-app-flutter
Version: v1.9.2
Project Flavor: None selected
___________________________________________________

Version is currently cached locally.

Cache Path: /Users/phil/fvm/versions/v1.9.2
Channel: false
SDK Version: 1.9.2

IDE Links
VSCode: .fvm/flutter_sdk
Android Studio: /Users/phil/Desktop/app-app-flutter/.fvm/flutter_sdk

Configured env paths:
___________________________________________________

Flutter:

Dart:
/usr/local/Cellar/dart/2.14.4/libexec/bin/dart

FVM_HOME:
not set
```

I'm running on macOS Big Sur and installed FVM via:  dart pub global activate fvm.

**Comment by leoafarias** [comment]
@callmephil this fix has been merged and will try to deploy it today

**Comment by msarkrish** [comment]
@leoafarias I'm facing this same problem in latest fvm 2.2.5 version.
Is there any solution for this problem.

**Comment by leoafarias** [comment]
@msarkrish there was a regression, pushing a fix on 2.2.6 in a few minutes

**Comment by msarkrish** [comment]
Thank you @leoafarias for immediate fix.
